### PR TITLE
Fix CI tests for recent PRs

### PR DESCRIPTION
**Pull request by karllark**
A couple of recent PRs changed the names of the blackbody (BB) components, fixed part of the BB setup, and put in a quick fix on the limits of the gas line fwhm values (see PR #176 and #177).   

This PR updates the tests so that they pass.  The M101 fits looks similar and fine.

### CODE REVIEW

**Comment by karllark** [comment]
Codacy error can be ignored.

File: pahfit/base.py [hunk 1]
```diff
@@ -363,27 +399,55 @@ def plot(self, axs, x, y, yerr, model, model_samples=1000, scalefac_resid=2):
...
-                              "Total Continuum Emissions",
-                              "Continuum Components"], prop={'size': 10}, loc="best", facecolor="white", framealpha=1,
-                  ncol=3)
+        ax.legend(
+            Leg_lines,
+            [
+                "S07_attenuation",
+                "Spectrum Fit",
+                "Dust Features",
+                r"Atomic and $H_2$ Lines",
```
Comment (els1) [inline pahfit/base.py:441]: Typo?   r"Atomic and $H_2$ Lines"

**Review by els1 — COMMENTED** [review]
Looks good to me.

File: pahfit/base.py (hunk 1 above)
Comment (karllark) [reply pahfit/base.py:441]: The "r" is in front to allow indicate that there is latex ahead in the string.  See https://www.pythontutorial.net/python-basics/python-raw-strings/.

Comment (els1) [reply pahfit/base.py:441]: Ah. Thanks!

**Pull request merged**